src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoccontentlocation/Example1.java

```diff
@@ -1,17 +1,23 @@
 /*xml
 <module name="Checker">
   <module name="TreeWalker">
-    <module name="JavadocContentLocationCheck"/>
+    <module name="JavadocContentLocation"/>
   </module>
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoccontentlocation;
+
+class Example1 {
+
 // xdoc section -- start
 /** This comment causes a violation because it starts from the first line
-  * and spans several lines.
-  */
+ * and spans several lines.
+ */
```

Comment:
please do me favor and add `private int filed1;` under this comment.

please add some simple fields for all other comments, it took me a while to understand why such comments matters.

Reply:
Sure thing.

Reply:
> please do me favor and add `private int filed1;` under this comment.
> 
> please add some simple fields for all other comments, it took me a while to understand why such comments matters.

done